microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md

```diff
@@ -149,15 +153,12 @@ if (sendMessageId) {
 
 ---
 
-
 <!-- ====================================================================== -->
 ## Step 4: Make your `stars.jpeg` available from any browser tab
 
-When you pass the `images/stars.jpeg`, you must use the `chrome.runtime.getURL` API (or `chrome.extension.getURL` if using Manifest V2) instead of just passing in the relative URL without the extra prefix like in the previous section.  That extra prefix is returned by `getUrl` with the image attached, and looks something like the following:
+To make `images/stars.jpeg` available from any browser tab, you must use the `chrome.runtime.getURL` API.  
```

Comment:
```suggestion
To make `images/stars.jpeg` available from any browser tab, you must use the `chrome.runtime.getURL` API.
```
two trailing spaces is significant in Markdown (linebreak) and this repo was cleaned of them